routers/web/repo/commit.go

```diff
@@ -83,7 +83,16 @@ func Commits(ctx *context.Context) {
 		return
 	}
 	ctx.Data["Commits"] = processGitCommits(ctx, commits)
-
+	commitIDs := make([]string, 0, len(commits))
+	for _, c := range commits {
+		commitIDs = append(commitIDs, c.ID.String())
+	}
+	commitsTagsMap, err := repo_model.FindTagsByCommitIDs(ctx, ctx.Repo.Repository.ID, commitIDs...)
+	if err != nil {
+		ctx.ServerError("CommitsByRange", err)
```

Comment:
```suggestion
		ctx.ServerError("FindTagsByCommitIDs", err)
```

Reply:
And should we break the page if we got error when finding tags?

Reply:
fac2eb8038284da4ac5aa8cc34f2841ff2aa781c